.circleci/config.yml

```diff
@@ -48,7 +48,6 @@ workflows:
           context: architect
           app_name: "azure-admission-controller"
           app_collection_repo: "azure-app-collection"
-          unique: "true"
```

Comment:
we don't need to specify this anymore?

Reply:
nope, this was removed in architect-orb 1.0.0 https://github.com/giantswarm/architect-orb/blob/master/CHANGELOG.md#removed-1